.github/workflows/e2e_api_tests.yml

```
@@ -69,6 +69,9 @@ jobs:
     tests:
         name: e2e_api tests
         runs-on: ubuntu-latest
+        strategy:
```

actionlint (1.7.4)

"strategy" section should not be empty. please remove this section if it's unnecessary

(syntax-check)